src/test/java/org/opensearch/security/sanity/tests/SingleClusterSanityIT.java

```diff
@@ -34,6 +38,13 @@ public void testSecurityPluginInstallation() throws Exception {
         verifyPluginInstallationOnAllNodes();
     }
 
+    @Test
+    public void testAdminCredentials_customStrongPassword_shouldSucceed() throws Exception {
```

Comment:
performRequest() requires an exception to be handled or thrown. I chose to not handle it since the test should fail if performRequest didn't work as expected.